code/parse/sexp.cpp

```diff
@@ -15821,6 +15832,18 @@ void sexp_set_debriefing_persona(int node)
 	The_mission.debriefing_persona = persona;
 }
 
+// MjnMixael
+void sexp_set_traitor_override(int node)
+{
+	SCP_string override = CTEXT(node);
+
+	if (!stricmp(override.c_str(), "none")) {
```

Comment:
Since the sexp argument list is populated using the token `SEXP_NONE_STRING`, that should be used here too.

Reply:
See `sexp_set_skybox_model_preload` or the `OPF_TEAM_COLOR` error check.